static/app/utils/profiling/renderers/gridRenderer.tsx

```diff
@@ -13,18 +13,20 @@ export function getIntervalTimeAtX(logicalSpaceToConfigView: mat3, x: number): n
   return Math.round(vector * 10) / 10;
 }
 
+//
```

Comment:
?

Reply:
I think I wanted to write a comment there but ended up just adding to the body comment... removing